crawler.py

```diff
@@ -332,8 +332,12 @@ def truncate(string, max_len):
 
 
 def debug_dump_site_html(name, html):
-    with open(f"debug-sites/site-{name}.html", "w") as test_log:
-        print(html, file=test_log)
+    dump_sites_path = Path("debug-sites")
+    dump_sites_path.mkdir(parents=True, exist_ok=True)
+    site_name = "site-{}.html".format(name)
+    site_debug_path = dump_sites_path / f"sites-{name}.html"
```

Comment:
I don't get this pair of lines. Is the first (`site_name = ...`) a leftover?

Reply:
I think you are right. This line is obsolete then.